### PR TITLE
refine enable tls for client/components

### PR DESCRIPTION
**Pull request by lysu**
### What is changed, added or deleted? (Required)

- split 《为 TiDB 组件间开启 TLS 和数据加密存储》as 《为 TiDB 组件间开启 TLS 》and 《为 TiDB 开启数据加密存储》
- use more union name and url  for enable client/components
- refine title selection in enable-tls-between-components.md
- misc refine

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/docs-cn/3325)

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: enable-encrypt-stored-data.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: 为 TiDB 开启数据加密存储
+category: how-to
+aliases: ['/docs-cn/dev/how-to/secure/enable-encrypt-stored-data/']
```
Comment (imtbkcat) [inline enable-encrypt-stored-data.md:4]: Please remove aliases for new document.

File: enable-tls-between-clients.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 ---
-title: 使用加密连接
+title: 为 TiDB 客户端服务端间通信开启加密传输
 category: how-to
-aliases: ['/docs-cn/dev/how-to/secure/enable-tls-clients/']
+aliases: ['/docs-cn/dev/how-to/secure/enable-tls-between-clients.md/']
```
Comment (imtbkcat) [inline enable-tls-between-clients.md:4]: dito

File: enable-encrypt-stored-data.md (hunk 1 above)
Comment (TomShawn) [inline enable-encrypt-stored-data.md:4]:
```suggestion
```

File: enable-encrypt-stored-data.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+## 开启数据加密存储
```
Comment (TomShawn) [inline enable-encrypt-stored-data.md:7]:
```suggestion
# 为 TiDB 开启数据加密存储
```

File: enable-encrypt-stored-data.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+在 TiDB 集群中，用户的数据都存储在 TiKV 中，配置了 TiKV 数据加密存储功能，就代表 TiDB 集群已经加密存储了用户的数据。本部分主要介绍如何配置 TiKV 的加密存储功能。
+
+### 操作流程
```
Comment (TomShawn) [inline enable-encrypt-stored-data.md:11]:
```suggestion
## 操作流程
```

File: enable-encrypt-stored-data.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    ./tikv-ctl random-hex --len 256 > cipher-file-256
+    ```
+
+    > **注意：**
+    >
+    > TiKV 只接受 hex 格式的 token 文件，文件的长度必须是 2^n，并且小于等于 1024。
```
Comment (TomShawn) [inline enable-encrypt-stored-data.md:25]:
```suggestion
    > TiKV 只接受 hex 格式的 token 文件，文件的长度必须是 2<sup>n</sup>，并且小于等于 1024。
```

File: enable-encrypt-stored-data.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+    ```toml
+    [security]
+    # Cipher file 的存储路径
+    cipher-file = "/path/to/cipher-file-256"
+    ```
+
+> **注意：**
+>
+> 若使用 [Lightning](/tidb-lightning/tidb-lightning-overview.md) 向集群导入数据，如果目标集群开启了加密功能，Lightning 生成的 sst 文件也必须是加密的格式。
```
Comment (TomShawn) [inline enable-encrypt-stored-data.md:37]:
```suggestion
> 若使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) 向集群导入数据，如果目标集群开启了加密功能，Lightning 生成的 SST 文件也必须是加密的格式。
```

File: enable-tls-between-clients.md (hunk 2 above)
Comment (TomShawn) [inline enable-tls-between-clients.md:4]:
```suggestion
aliases: ['/docs-cn/dev/how-to/secure/enable-tls-clients/','/docs-cn/dev/encrypted-connections-with-tls-protocols/']
```

File: enable-tls-between-clients.md [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
 ---
 
 # 使用加密连接
```
Comment (TomShawn) [inline enable-tls-between-clients.md:7]:
```suggestion
# 为 TiDB 客户端服务端间通信开启加密传输
```

File: enable-tls-between-components.md [hunk 8]
```diff
@@ -1,27 +1,25 @@
 ---
-title: 为 TiDB 组件间开启 TLS 和数据加密存储
+title: 为 TiDB 组件间通信开启加密传输
 category: how-to
 aliases: ['/docs-cn/dev/how-to/secure/enable-tls-between-components/']
 ---
 
-# 为 TiDB 组件间开启 TLS 和数据加密存储
+## 支持使用加密传输的组件
```
Comment (TomShawn) [inline enable-tls-between-components.md:7]: 一级标题需要与 metadata 部分的标题一致。
```suggestion
## 为 TiDB 组件间通信开启加密传输
```

File: enable-encrypt-stored-data.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+### 使用限制
```
Comment (TomShawn) [inline enable-encrypt-stored-data.md:39]:
```suggestion
## 使用限制
```

**Comment by jackysp** [comment]
Please address comments @lysu

**Comment by lysu** [comment]
@imtbkcat @TomShawn done, PTAL thx

File: enable-encrypt-stored-data.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
+---
+title: 为 TiDB 开启数据加密存储
```
Comment (TomShawn) [inline enable-encrypt-stored-data.md:2]:
```suggestion
title: 为 TiDB 开启数据加密存储
summary: 介绍如何为 TiDB 开启数据加密存储。
```

**Pull request merged**